### PR TITLE
feat(dist): update cargo dist and add npm installer

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -18,28 +52,33 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install Rust
-        run: rustup update 1.67.1 --no-self-update && rustup default 1.67.1
+        run: rustup update stable --no-self-update && rustup default stable
```
Comment (Gankra) [inline .github/workflows/release.yml:55]: did you do this on purpose?

File: Cargo.toml [hunk 2]
```diff
@@ -50,9 +50,21 @@ assert_fs = "1.0.7"
 inherits = "release"
 lto = "thin"
 
+# Config for 'cargo dist'
 [workspace.metadata.dist]
-cargo-dist-version = "0.0.5"
+# The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.0.6-prerelease.8"
+# The preferred Rust toolchain to use in CI (rustup toolchain syntax)
 rust-toolchain-version = "stable"
```
Comment (Gankra) [inline Cargo.toml:58]: oh huh. pre-existing

**Review by Gankra — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (ashleygwilliams) [reply .github/workflows/release.yml:55]: i did- mostly because i knew i would forget to and wanted to pin to stable

File: Cargo.toml (hunk 2 above)
Comment (ashleygwilliams) [reply Cargo.toml:58]: yeah i had edited it

**Pull request merged**